metadata/edgex-ui.md

```diff
@@ -1,3 +1,23 @@
-A graphical user interface for EdgeX Foundry
+EdgeX UI
 ---
-The EdgeX UI is for demonstration and developer use to manage and monitor a single instance of EdgeX Foundry
\ No newline at end of file
+EdgeX UI is a graphical user interface for EdgeX.
+It is intended for demonstration and development use to manage and monitor a
+single instance of EdgeX Foundry.
+
+For a command-line interface, see https://snapcraft.io/edgex-cli
```

Comment:
For a graphical user interface, see https://snapcraft.io/edgex-ui

Reply:
This is the UI's description, redirecting users the CLI is that's what they prefer.